hackathon_site/hardware/serializers.py

```diff
@@ -80,6 +80,25 @@ class Meta:
             "part_returned_health",
         )
 
+class OrderItemListSerializer(serializers.ModelSerializer):
+    team_code = serializers.SerializerMethodField()
+    order_id = serializers.SerializerMethodField()
+
+    class Meta:
+        model = OrderItem
+        fields = (
+            "order_id"
+            "team_code"
```

Comment:
Looking at the Figma, you should also add created_at, updated_at , part_returned_health and hardware to the fields list